libraries/chain/include/eosio/chain/subjective_billing.hpp

```diff
@@ -107,7 +109,7 @@ class subjective_billing {
    void disable_account( chain::account_name a ) { _disabled_accounts.emplace( a ); }
    bool is_account_disabled(const chain::account_name& a ) const { return _disabled || _disabled_accounts.count( a ); }
 
-   void subjective_bill( const chain::transaction_id_type& id, const fc::time_point& expire,
+   void subjective_bill( const chain::transaction_id_type& id, const fc::time_point_sec& expire,
```

Comment:
`trx_cache_entry` stores the trx expiration as a `time_point`. This could instead be a `time_point_sec` saving some space. I didn't make the change as it likely would not be a benefit as it currently has 64bit alignment. I'm open to changing it to a `time_point_sec` if that would make the code cleaner.

Reply:
It looks like getting a value for `expire` is more appropriate here than a `const&`. First it is usually constructed, and second `fc::time_point_sec` is only 8 bytes.

Reply:
Done.